### PR TITLE
FLPATH-473:set additional info for report urls

### PR DESCRIPTION
**Pull request by RichardW98**
**What this PR does / why we need it**:
add additional info for report links
**Which issue(s) this PR fixes** :
Fixes #473
Screenshot 2023-06-19 at 12 28 39 PM

example workflowResponse:
```
[
  {
    "workFlowExecutionId": "30f5b124-db7f-463d-92a6-575457ef2cfb",
    "projectId": "79d23d8c-8e33-409d-b786-8c908e330182",
    "workFlowName": "pendingWithAlertMessageWorkFlow",
    "workStatus": "PENDING",
    "startDate": "2023-06-19 12:46:00.842",
    "endDate": "2023-06-19 12:46:00.879",
    "executeBy": "test",
    "additionalInfos": [
      {
        "key": "Alert message",
        "value": "http://localhost:8080"
      },
      {
        "key": "DoNothing",
        "value": null
      }
    ]
  }
]
```
**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Unit tests
- [ ] Integration tests
- [ ] CI
- [ ] Documentation
- [ ] Auto-generated SDK code

**Impacted services**
- [ ] Workflow Service
- [ ] Notification Service

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

File: prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/CreateApplicationTask.java [hunk 1]
```diff
@@ -71,6 +71,7 @@ else if (result instanceof Result.Success<App> success) {
 			workContext.put("application", success.value());
 			taskLogger.logInfoWithSlf4j("MTA client returned success result for application creation: {}",
 					success.value().name());
+			addAdditionInfo("VCS branch", repo);
```
Comment (pkliczewski) [inline prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/CreateApplicationTask.java:74]: What VCS is standing for?

**Review by pkliczewski — COMMENTED** [review]
overall looks good

File: prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/CreateApplicationTask.java (hunk 1 above)
Comment (RichardW98) [reply prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/CreateApplicationTask.java:74]: version control system

**Review by masayag — APPROVED** [review]

**Pull request merged**